conda/meta.yaml

```diff
@@ -38,13 +38,13 @@ test:
 
 requirements:
   host:
-    - python
+    - python {{ python }}
     - pip
     - setuptools_scm
     - setuptools
     - pytest-runner
```

Comment:
I think you can remove `pytest-runner`. Also consider adding a "build" section here on the requirements with the content similar to the one I suggest on the confluence page. It is not strictly necessary but it is a good future-proof mechanism for pinning versions..

Reply:
I'll remove pytest-runner. The build section is there already. See line 11.